modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -277,89 +382,218 @@ export default function EditNotificationTemplate({
 
 						<div className="col-lg-6 lfr__notification-template-card">
 							<Card title={Liferay.Language.get('settings')}>
-								<InputLocalized
-									label={Liferay.Language.get('to')}
-									name="to"
-									onChange={(translation) => {
-										setValues({
-											...values,
-											to: translation,
-										});
-									}}
-									placeholder=""
-									selectedLocale={selectedLocale}
-									translations={values.to}
-								/>
-
-								<div className="row">
-									<div className="col-lg-6">
-										<Input
-											label={Liferay.Language.get('cc')}
-											name="cc"
-											onChange={({target}) =>
-												setValues({
-													...values,
-													cc: target.value,
-												})
-											}
-											value={values.cc}
-										/>
-									</div>
-
-									<div className="col-lg-6">
-										<Input
-											label={Liferay.Language.get('bcc')}
-											name="bcc"
-											onChange={({target}) =>
-												setValues({
-													...values,
-													bcc: target.value,
-												})
-											}
-											value={values.bcc}
-										/>
-									</div>
-								</div>
-
-								<div className="row">
-									<div className="col-lg-6">
-										<Input
-											error={errors.from}
+								{(Liferay.FeatureFlags['LPS-162133'] &&
+									notificationTemplateType ===
+										'userNotification') ||
+								values.type === 'userNotification' ? (
+									<>
+										<SingleSelect
 											label={Liferay.Language.get(
-												'from-address'
+												'recipients'
 											)}
-											name="fromAddress"
-											onChange={({target}) =>
+											onChange={(item) => {
 												setValues({
 													...values,
-													from: target.value,
-												})
+													recipientType: item.value,
+												});
+
+												if (item.value === 'role') {
+													getAccountRoles('');
+												}
+											}}
+											options={RECIPIENT_OPTIONS}
+											value={
+												RECIPIENT_OPTIONS.find(
+													(recipient) =>
+														values.recipientType ===
+														recipient.value
+												)?.label
```

Comment:
Hey @alinedoleron, there is a small bug here, when we try to change the `value` it is not changing visually, you can fix this in the `BaseSelect.tsx` component by adding the value as prop to `ClayAutocomplete.Input `on line `88`.

```tsx
<ClayAutocomplete.Input
    defaultValue={value}
    disabled={disabled}
    onClick={() => setDropdownActive((active) => !active)}
    placeholder={placeholder}
    ref={inputRef}
    value={value}
    {...restProps}
/>
```